scripts/check-sprint-last-day.py

```diff
@@ -0,0 +1,129 @@
+import requests
+import os
+import sys
+from datetime import datetime, timedelta
+
+
+GITHUB_GRAPHQL_URL = "https://api.github.com/graphql"
+
+
+def get_github_project_info(github_token, github_org, github_project):
+    headers = {
+        "Authorization": f"Bearer {github_token}",
+        "Content-Type": "application/json"
+    }
+    query = '''
+    {
+      organization(login: "%s") {
+        projectsV2(first: 20) {
+          nodes {
+            id
+            title
+            number
+          }
+        }
+      }
+    }
+    ''' % (github_org)
+    payload = {
+        "query": query
+    }
+
+    response = requests.post(GITHUB_GRAPHQL_URL, headers=headers, json=payload)
+    if response.status_code == 200:
+        # fine project by title
+        nodes = response.json().get("data").get("organization").get("projectsV2").get("nodes")
+        for node in nodes:
+            if node.get("title") == github_project:
+                return node
+    else:
+        response.raise_for_status()
+
+
+def get_current_sprint(github_token, project_id):
+    headers = {
+        "Authorization": f"Bearer {github_token}",
+        "Content-Type": "application/json"
+    }
+    query = '''
+    query {
+      node(id: "%s") {
+        ... on ProjectV2 {
+          fields(first: 20) {
+            nodes {
+              ... on ProjectV2IterationField {
+                configuration {
+                  iterations {
+                    startDate
+                    id
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+    ''' % (project_id)
+
+    payload = {
+        "query": query
+    }
+
+    response = requests.post(GITHUB_GRAPHQL_URL, headers=headers, json=payload)
+    if response.status_code == 200:
+        # fine project by title
+        result = response.json().get("data").get("node").get("fields").get("nodes")
+        filtered_result = [node for node in result if 'configuration' in node]
+        iterations = filtered_result[0].get("configuration").get("iterations")
+
+        # Find current iteration
+        current_date = datetime.now().date()
+        current_iteration = None
+        for iteration in iterations:
```

Comment:
nit: Do we know if the results are ordered by creation time? If so, we probably only need to loop from the last a few items to find the current sprint, rather than iterating from the beginning